examples/ad-lightbox.amp.html

```diff
@@ -0,0 +1,103 @@
+<!doctype html>
+<html ⚡>
+<head>
+    <meta charset="utf-8">
+    <title>AMP #0</title>
+    <link rel="canonical" href="amps.html" >
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <script async custom-element="amp-lightbox" src="https://cdn.ampproject.org/v0/amp-lightbox-0.1.js"></script>
+    <script async custom-element="amp-carousel" src="https://cdn.ampproject.org/v0/amp-carousel-0.1.js"></script>
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <style type="text/css">
+      html, body {
+        height: 100% !important;
+      }
+      body {
+        font-family: sans-serif;
+      }
+      amp-ad-banner {
+        display: block;
+        width: 100%;
+        height: 100%; /* TODO(alanorozco): make height dynamic */
+        position: relative;
+        background: #ccc;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+      }
+      .main-lightbox {
+        background: #000;
+        color: #ccc;
+      }
+      .button {
+        cursor: pointer;
+        display: inline-block;
+      }
+      .close-button {
+        position: absolute;
+        right: 10px;
+        top: 10px;
+        font-size: 14px;
+        border-radius: 20px;
+        width: 30px;
+        height: 30px;
+        border: 1px solid #ccc;
+        text-align: center;
+        line-height: 30px;
+      }
+      .lightbox-header {
+        height: 50px;
+      }
+    </style>
+</head>
+<body>
+  <!--
+    THIS EXAMPLE IS INVALID AMP.
+
+    It corresponds to an in-progress prototype. It will not validate or function
+    properly under normal circumstances.
+  -->
+
+  <amp-ad-banner layout="fill">
```

Comment:
`amp-ad-banner` is unimplemented, so it's not needed. The document is not supposed to validate :)